### PR TITLE
 python3Packages.cachecontrol: 0.12.6 -> 0.12.8 

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Update to latest upstream release 0.12.8

- Enable tests

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/cachecontrol/default.nix [hunk 1]
```diff
@@ -1,34 +1,46 @@
...
-, pytest
+, pytestCheckHook
+, requests
 }:
 
 buildPythonPackage rec {
-  version = "0.12.6";
-  pname = "CacheControl";
+  version = "0.12.8";
+  pname = "cachecontrol";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/cachecontrol/default.nix:14]:
```suggestion
  pname = "cachecontrol";
  version = "0.12.8";
```

File: pkgs/development/python-modules/cachecontrol/default.nix [hunk 2]
```diff
@@ -1,34 +1,46 @@
...
+    "cachecontrol"
+  ];
 
   meta = with lib; {
-    homepage = "https://github.com/ionrock/cachecontrol";
     description = "Httplib2 caching for requests";
+    homepage = "https://github.com/ionrock/cachecontrol";
     license = licenses.asl20;
-    maintainers = [ maintainers.costrouc ];
+    maintainers = with maintainers; [ costrouc ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/cachecontrol/default.nix:44]: That maintainer is currently not that active and their are a lot of packages under the name. Maybe we should take some away?

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 143986` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages marked as broken and skipped:</summary>
  <ul>
    <li>cwltool</li>
    <li>python38Packages.schema-salad</li>
    <li>python38Packages.tvdb_api</li>
    <li>python39Packages.schema-salad</li>
    <li>python39Packages.tvdb_api</li>
    <li>tvnamer</li>
  </ul>
</details>
<details>
  <summary>70 packages built:</summary>
  <ul>
    <li>cmake-language-server</li>
    <li>cryptop</li>
    <li>home-assistant</li>
    <li>ioccheck</li>
    <li>lexicon</li>
    <li>mnamer</li>
    <li>poetry2conda (python39Packages.poetry2conda)</li>
    <li>python38Packages.aioeafm</li>
    <li>python38Packages.aiogithubapi</li>
    <li>python38Packages.aiomodernforms</li>
    <li>python38Packages.aria2p</li>
    <li>python38Packages.backoff</li>
    <li>python38Packages.cachecontrol</li>
    <li>python38Packages.censys</li>
    <li>python38Packages.geocachingapi</li>
    <li>python38Packages.hypothesis-auto</li>
    <li>python38Packages.mapbox</li>
    <li>python38Packages.myjwt</li>
    <li>python38Packages.poetry</li>
    <li>python38Packages.poetry2conda</li>
    <li>python38Packages.pyairnow</li>
    <li>python38Packages.pyiqvia</li>
    <li>python38Packages.pyopenuv</li>
    <li>python38Packages.pyprosegur</li>
    <li>python38Packages.pysnow</li>
    <li>python38Packages.pysychonaut</li>
    <li>python38Packages.python-miio</li>
    <li>python38Packages.questionary</li>
    <li>python38Packages.requests-cache</li>
    <li>python38Packages.scikit-bio</li>
    <li>python38Packages.simplisafe-python</li>
    <li>python38Packages.teslajsonpy</li>
    <li>python38Packages.toonapi</li>
    <li>python38Packages.url-normalize</li>
    <li>python38Packages.velbus-aio</li>
    <li>python38Packages.versionfinder</li>
    <li>python38Packages.wled</li>
    <li>python38Packages.yalesmartalarmclient</li>
    <li>python39Packages.aioeafm</li>
    <li>python39Packages.aiogithubapi</li>
    <li>python39Packages.aiomodernforms</li>
    <li>python39Packages.aria2p</li>
    <li>python39Packages.backoff</li>
    <li>python39Packages.cachecontrol</li>
    <li>python39Packages.censys</li>
    <li>python39Packages.geocachingapi</li>
    <li>python39Packages.hypothesis-auto</li>
    <li>python39Packages.mapbox</li>
    <li>python39Packages.myjwt</li>
    <li>python39Packages.poetry</li>
    <li>python39Packages.pyairnow</li>
    <li>python39Packages.pyiqvia</li>
    <li>python39Packages.pyopenuv</li>
    <li>python39Packages.pyprosegur</li>
    <li>python39Packages.pysnow</li>
    <li>python39Packages.pysychonaut</li>
    <li>python39Packages.python-miio</li>
    <li>python39Packages.questionary</li>
    <li>python39Packages.requests-cache</li>
    <li>python39Packages.scikit-bio</li>
    <li>python39Packages.simplisafe-python</li>
    <li>python39Packages.teslajsonpy</li>
    <li>python39Packages.toonapi</li>
    <li>python39Packages.url-normalize</li>
    <li>python39Packages.velbus-aio</li>
    <li>python39Packages.versionfinder</li>
    <li>python39Packages.wled</li>
    <li>python39Packages.yalesmartalarmclient</li>
    <li>tebreak</li>
    <li>theharvester</li>
  </ul>
</details>

File: pkgs/development/python-modules/cachecontrol/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/development/python-modules/cachecontrol/default.nix]: Updated, thanks.

**Pull request merged**